ui/Screen/Settings.svelte

```diff
@@ -2,52 +2,79 @@
   import {
     clear,
     clearCache,
+    fetchSeeds,
     settings,
     updateAppearance,
+    updatePeerConfig,
     updateRegistry,
   } from "../src/session.ts";
   import { networkOptions, themeOptions } from "../src/settings.ts";
 
-  import { Title, Text, Button } from "../DesignSystem/Primitive";
+  import { Button, Input, Text, Title } from "../DesignSystem/Primitive";
   import { SidebarLayout, SegmentedControl } from "../DesignSystem/Component";
 
+  let seeds = fetchSeeds();
+
   const updateNetwork = event =>
     updateRegistry({ ...$settings.registry, network: event.detail });
 
   const updateTheme = event =>
     updateAppearance({ ...$settings.appearance, theme: event.detail });
+
+  const updateSeeds = event => {
+    updatePeerConfig(event.target.value);
+  };
 </script>
 
 <style>
   .container {
     max-width: var(--content-max-width);
-    margin: 64px auto;
+    margin: 64px 100px;
```

Comment:
This doesn't centre the content anymore. Don't understand what you were trying to do here.